### PR TITLE
Update Artman YAMLs.

### PR DESCRIPTION
**Pull request by lukesneeringer**
Companion to googleapis/artman#162.

### CODE REVIEW

File: gapic/api/artman_bigtable.yaml [hunk 1]
```diff
@@ -6,46 +6,65 @@ common:
...
+      gapic_subpath: google-cloud-bigtable-v2/cloud.google.com/go/bigtable/apiv2
 java:
-  gapic_code_dir: ${REPOROOT}/google-cloud-java/google-cloud-bigtable
-  git_repo:
-    gapic_subpath: generated/java/google-cloud-bigtable-v2
+  gapic_code_dir: ${REPOROOT}/artman/output/java/google-cloud-bigtable
+  git_repos:
+    java:
+      gapic_component: src/main/java/com/google/cloud/bigtable/spi/v2
+      gapic_subpath: google-cloud-pubsub/src/main/java/com/google/cloud/bigtable/spi/v2
```
Comment (garrettjonesgoogle) [inline gapic/api/artman_bigtable.yaml]: * The value that I see here in `gapic_component` is generated inside toolkit for java; this doesn't need to be set at all.
* `gapic_subpath` should just be `google-cloud-pubsub`

File: gapic/api/artman_bigtable.yaml [hunk 2]
```diff
@@ -6,46 +6,65 @@ common:
...
-  git_repo:
-    location: git@github.com:GoogleCloudPlatform/google-cloud-ruby.git
-    gapic_component: lib/google/cloud/bigtable/v2
-    gapic_subpath: google-cloud-bigtable/lib/google/cloud/bigtable/v2
+  gapic_code_dir: ${REPOROOT}/artman/output/ruby/google-cloud-bigtable
+  git_repos:
+    ruby:
+      location: git@github.com:GoogleCloudPlatform/google-cloud-ruby.git
+      gapic_component: lib/google/cloud/bigtable/v2
+      gapic_subpath: google-cloud-bigtable/lib/google/cloud/bigtable/v2
```
Comment (garrettjonesgoogle) [inline gapic/api/artman_bigtable.yaml]: Why the redundancy between `gapic_component` and `gapic_subpath`?

Comment (lukesneeringer) [inline gapic/api/artman_bigtable.yaml]: How this is used conceptually is "cp `{generated code}/{component} -rf {destination repo}/{subpath}".

Even though a lot of it looks duplicative, there is an important distinction. In Node and Ruby, the GAPIC gets "tucked" inside of an existing manual library. Since this process involves removing everything at `subpath` before copying the new components, this staging process would effectively delete the manual layer completely.

Additionally, in Node and Ruby, only a subset of the generated code is used in these cases (the package metadata and outer `index` files are thrown away). However, the entire generated code is used in GAPIC only (so the settings would be distinct for a GAPIC only library).

Basically: this is not _actually_ redundant; there is a specific semantic meaning, which is to copy only this slice of the GAPIC into this particular slice of the target repo.

File: gapic/api/artman_bigtable.yaml (hunk 1 above)
Comment (lukesneeringer) [inline gapic/api/artman_bigtable.yaml]: Okay, I got that wrong for Java. :-) Quick question: Would you check on the natural language ones and tell me what the correct values are there? (See my explanation of the values in your other comment for details.) With the settings you are asking for, the side-by-side case would end up stomping over itself.

File: gapic/api/artman_language_v1.yaml [hunk 3]
```diff
@@ -6,46 +6,65 @@ common:
...
+      gapic_subpath: google-cloud-language-v1/cloud.google.com/go/language/apiv1
 java:
-  gapic_code_dir: ${REPOROOT}/google-cloud-java/google-cloud-language
-  git_repo:
-    gapic_subpath: generated/java/google-cloud-language-v1
+  gapic_code_dir: ${REPOROOT}/artman/output/java/google-cloud-language
+  git_repos:
+    java:
+      gapic_component: src/main/java/com/google/cloud/language/spi/v1
+      gapic_subpath: google-cloud-pubsub/src/main/java/com/google/cloud/language/spi/v1
```
Comment (garrettjonesgoogle) [inline gapic/api/artman_language_v1.yaml]: This should be `google-cloud-language` (it looks like you have `google-cloud-pubsub` in a number of places)

File: gapic/api/artman_bigtable.yaml (hunk 1 above)
Comment (garrettjonesgoogle) [inline gapic/api/artman_bigtable.yaml]: The side-by-side case will still work fine for Java.

* In the google-cloud repo, the existing code is not removed before new code is copied in, so there is no problem
* In the staging repo, the existing code is removed, but the roots are distinct, so they won't stomp over each other

File: gapic/api/artman_language_v1.yaml (hunk 3 above)
Comment (lukesneeringer) [inline gapic/api/artman_language_v1.yaml]: Thanks. (I used pubsub as my copy / paste template. Eek!)

File: gapic/api/artman_bigtable.yaml (hunk 1 above)
Comment (lukesneeringer) [inline gapic/api/artman_bigtable.yaml]: > In the google-cloud repo, the existing code is not removed before new code is copied in, so there is no problem

`--staging=local` very much removes what used to be at that spot ([here][1]). This is also true of the Artman [_status quo ante_][2] before my refactor last week.

  [1]: https://github.com/googleapis/artman/blob/master/artman/tasks/publish/local.py#L59
  [2]: https://github.com/googleapis/artman/blob/e45376ffd4eb6d3d9f9f3bdac5706cea9adf4d70/pipeline/tasks/gapic_tasks.py#L120

That said, my feeling on the language specific segments of these files is that the language owners decide. Therefore, I will have this changed to what you are asking momentarily. If it turns out it is not right for you, you are always free to change it (and you very much do not need my buy-in to do so). :-)

Comment (garrettjonesgoogle) [inline gapic/api/artman_bigtable.yaml]: Maybe we have a different understanding of staging? For me, it is 1:1 with api-client-staging, and putting code into google-cloud-java isn't called "staging".

File: gapic/api/artman_language_v1.yaml (hunk 3 above)
Comment (lukesneeringer) [inline gapic/api/artman_language_v1.yaml]: Just grepped every non-pubsub file to ensure I got them all. Thank you!

**Comment by lukesneeringer** [comment]
@garrettjonesgoogle I believe I updated all the Java YAML segments to what you are expecting. Let me know if what is there looks right for your workflow.
